creator-onboarding/readme/artist-onboarding-steps.md

```diff
@@ -1,82 +1,96 @@
 ---
 order: 1000
 ---
-# Artist Onboarding Steps
 
-Below you'll find an overview of each step of our onboarding process as well as an **estimated** time required for each step.
+Below, you'll find an overview of each step of our onboarding process, as well as an **, estimated** time required for each step.
 
 ## 1. Application
 
-Applications will remain open indefinitely. You may apply using this [application form](https://mpphuqcai14.typeform.com/apply-ab-artist). We expect that artists have a creative history as well as the ability to provide an original generative script. We process applications in chronological order, prioritizing those with working prototypes. If you plan to work in a collaborative fashion, please provide complete information for all contributors. All submitted work must be original. We do not intend to release projects that use licensed, purchased or recycled content. Please make sure you’re bringing your own unique project to Art Blocks.
+For the time being, applications will remain open. You may apply using this [link](https://www.artblocks.io/apply). We expect that artists have a creative history and the ability to provide an original generative script. If you plan to work collaboratively, please provide complete information for all contributors. All submitted work must be original. We do not intend to release projects that use licensed, purchased, or recycled content. Please make sure you’re bringing your unique project to Art Blocks. If you’ve applied after October 28, 2022, you’ll be given a staging shell to deploy your script on testnet.
 
-To learn more about the application process please visit artblocks.io/learn.
+To learn more about the application process please visit [apply.artblocks.io](apply.artblocks.io).
 
 !!!info Timeline
-Due to the surge in Art Blocks popularity, applications may take up to several months to process.
+Due to high volume, applications may take up to several months to process. We appreciate your patience every step of the way. Please understand that Art Blocks operates using a mentorship model, providing hands-on assistance and guidance while managing hundreds of artists in our pipeline.
 !!!
 
-## 2. Script test runs and project review
+## 2. Upload to testnet
 
-Our team will review each submission and reach out to you to get an updated version of your working prototype when it's your turn in the queue.
+You'll be asked to upload your project to the artist staging site on the Goerli test network. Here you can mint test outputs of your project and work out any kinks in your script before the Art Blocks selection committee reviews it. 
```

Comment:
Nice to see the Ropsten -> Goerli changes. @madpinney would you be willing to audit all the AB Docs to verify this has been updated everywhere?